### PR TITLE
Method Seurat

### LINKED ISSUE #34: method_nonSpatial_Seurat

### PR DESCRIPTION
**Pull request by shdam**
Seurat method implemented with 1 config per method setting.

I couldn't find a way to use a custom neighborhood graph, so I chose to use Seurat::FindNeighbors (igraph SNN).

Closes #34

### CODE REVIEW

File: method/seurat/seurat.yml [hunk 1]
```diff
@@ -0,0 +1,319 @@
+name: seurat_env
```
Comment (niklasmueboe) [inline method/seurat/seurat.yml:1]: name is not necessary and can be removed

File: method/seurat/seurat.yml [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+  - xorg-renderproto=0.11.1
+  - xorg-xextproto=7.3.0
+  - xorg-xproto=7.0.31
+  - xz=5.2.6
+  - yaml=0.2.5
+  - yq=3.2.3
+  - zlib=1.2.13
+  - zstd=1.5.5
+  - pip:
+      - numpy==1.26.2
```
Comment (niklasmueboe) [inline method/seurat/seurat.yml:319]: Is there a reason why numpy is not installed via conda

**Review by niklasmueboe — CHANGES_REQUESTED** [review]

**Comment by niklasmueboe** [comment]
I think Seurat shouldn't use the neighborhood graph anyway because it serves as "non-spatial" control.

**Comment by shdam** [comment]
Seeing your stLearn comment, I will also tune k instead of providing resolution in the config.

**Comment by niklasmueboe** [comment]
Oh, good. Must have missed it here

**Comment by shdam** [comment]
Resolution tuning implemented

**Pull request merged**